package/utils/busybox/Config-defaults.in

```diff
@@ -1728,16 +1728,16 @@ config BUSYBOX_DEFAULT_SETSID
 	default n
 config BUSYBOX_DEFAULT_SWAPON
 	bool
-	default n
+	default y
```

Comment:
Please check how much the size of the busybox binary increases when you activate these options on a common target.

Reply:
No increase in binary size. Checked for ramips/AR.